layouts/_default/hackerstoolbox_schedule.html

```diff
@@ -0,0 +1,14 @@
+{{ define "main" }}
+{{ $hs := index $.Site.Data.hackers_toolbox
+(partial "projects/get_project.html" (dict "projectName" "hackers_toolbox")) }}
+<main class="container main">
+  <div class="events hs col-12 col-md-10 offset-md-1">
+    <h1>Hackers Toolbox Schedule (AY{{ $.Site.Params.academicYear }} Semester {{ $.Site.Params.semester }})</h1>
```

Comment:
I guess this section will automatically update to `AY2324 Semester 1` once the semester starts?

![image](https://github.com/nushackers/nushackers-site/assets/24467184/454c5c02-adbb-4f12-81cf-8b9149042a2c)

Reply:
we are keeping track of the current academic year and semester in `config.toml`, once that is updated data will be pulled from data yaml files corresponding to the current semester